airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -507,6 +552,11 @@ def __init__(self, **kwargs):
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs):
         return f"checkout/sessions/{stream_slice['checkout_session_id']}/line_items"
 
+    def stream_slices(
+        self, *, sync_mode: SyncMode, cursor_field: List[str] = None, stream_state: Mapping[str, Any] = None
+    ) -> Iterable[Optional[Mapping[str, Any]]]:
+        return [{}]
```

Comment:
nit: this (sort-of) no-op being repeated seems less than ideal, if it's necessary could abstract it to a Mixin?

Reply:
Good idea. Done